connect_four.py

```diff
@@ -111,44 +116,52 @@ def __init__(self) -> None:
         self.board = ConnectFourBoard()  #TODO add capability to change board size and win requirement
 
     def __game_won(self, player: str):
```

Comment:
wait this method doesnt return anything?

Reply:
Yeah as of rn it doesn't return anything. I wasn't sure if I needed it to return anything eventually or not. I will mark it -> None for now.